src/tests/ftest/deployment/network_failure.py

```diff
@@ -6,6 +6,7 @@
 import os
 import time
 from ClusterShell.NodeSet import NodeSet
+from collections import defaultdict
```

Comment:
We should resolve this pylint error by moving the import
```
import time
from collections import defaultdict
from ClusterShell.NodeSet import NodeSet
```

Reply:
Done